scipy/optimize/tests/test_optimize.py

```diff
@@ -2200,6 +2200,14 @@ def func(z, *params):
                           match=r'Either final optimization did not succeed'):
             optimize.brute(func, self.rranges, args=self.params, disp=True)
 
+    def test_coerce_args_param(self):
+        # optimize.brute should coerce non-iterable args to a tuple.
+        def f(x, *args):
+            return x ** args
```

Comment:
```suggestion
            return x ** args[0]
```
Although this works because `x` is an array, it seems unusual (conceptually) to raise `x` to the power of a tuple.